test/test-tarantool/call.test.py

```diff
@@ -0,0 +1,216 @@
+"""
+Tarantool's test box-py/call.test.py had a number of problem fixed in commits:
+72f8b89332b552cf20b2143e6dae7fe5042a9f79 ("Update tarantool-python to 0.7.1-1-g8847b8c"),
+110f681ac0a56e3016e9a95eb1db544e353fa812 ("python3: use string byte literal") and
+d421020eeb1b3c2ec66a49a3be0e8a363dbe8d3d ("python3: fix printing of called box command")
```

Comment:
The first one is correct, but reverting of the second and third commits does not break the test case.

Reply:
Fixed

Reply:
Reverting of those two commits still doesn't fail the test for me:

```
$ git checkout origin/ligurio/add-unittest # 9087358a6a1653a8cda450d5c884ce41aabaa1d0
$ git revert 110f681ac0a56e3016e9a95eb1db544e353fa812
$ git revert d421020eeb1b3c2ec66a49a3be0e8a363dbe8d3d
$ make test
<...>
[008] test-tarantool/call.test.py                                     [ pass ]
<...>
```

Reply:
I have reverted d421020eeb1b3c2ec66a49a3be0e8a363dbe8d3d and run call.test.py in test-run and box-py/call.test.py in tarantool repo. Both tests passed.

Seems the real fix for `box-py/call.test.py` was done in commit for tarantool-python (https://github.com/tarantool/tarantool-python/pull/186/commits/13b8dfd83be9d87e196bfefc909dbdfc4362192e) and in commit 45972c99a5607b74984edcdfdab5fd2a79c78eba "test: make output of box-py/call.test.py deterministic" (https://github.com/tarantool/tarantool/commit/45972c99a5607b74984edcdfdab5fd2a79c78eba)
and commit to tarantool-python is a single reason to keep that regression test for test-run.

Reply:
Sure. But its commit hash in tarantool-python is https://github.com/tarantool/tarantool-python/commit/8847b8c8bd1092d87601987d858e3cb9ff54f9f6 (13b8dfd83be9d87e196bfefc909dbdfc4362192e is from the [PR #186](https://github.com/tarantool/tarantool-python/pull/186), it was rebased at pushing to master).

Reply:
True, thanks for pointing to this!
```
commit 67bfb309c23b8e35aecaa8ea33189fbcac5fe19f
Author: Sergey Bronnikov <sergeyb@tarantool.org>
Date:   Fri Mar 26 10:29:07 2021 +0300

    test: add regression test for box-py/call.test.py
    
    Added test that covers fix in tarantool-python library:
    8847b8c8bd1092d87601987d858e3cb9ff54f9f6
    ("python3: make json.dumps compatible with Python 2") [1].
    
    1. https://github.com/tarantool/tarantool-python/pull/186/commits/8847b8c8bd1092d87601987d858e3cb9ff54f9f6

```

Reply:
The test file still refers 13b8dfd83be9d87e196bfefc909dbdfc4362192e. I'll update it.

Reply:
Updated.